### PR TITLE
CLI exceptions show up as readable return values instead of backtrace.

### PR DESCRIPTION
**Pull request by elorest**
### Description of the Change

Makes error messages look better by using CLI `exit!` command.

### CODE REVIEW

File: src/amber/cli/commands/database.cr [hunk 1]
```diff
@@ -89,7 +87,7 @@ module Amber::CLI
           Micrate::DB.connection_url = url.gsub(path, "/#{uri.scheme}")
           return path.gsub("/", "")
         else
-          raise "could not determine database name"
+          raise "Could not determine database name"
```
Comment (eliasjpr) [inline src/amber/cli/commands/database.cr]: Use exit here as well

File: src/amber/cli/commands/encrypt.cr [hunk 2]
```diff
@@ -19,6 +19,10 @@ module Amber::CLI
         encrypted_file = "config/environments/.#{env}.enc"
         unencrypted_file = "config/environments/#{env}.yml"
 
+        unless File.exists?(unencrypted_file) || File.exists?(encrypted_file)
+          raise Exceptions::Environment.new("./config/environments/", env)
```
Comment (eliasjpr) [inline src/amber/cli/commands/encrypt.cr]: Exit?

File: src/amber/cli/commands/database.cr (hunk 1 above)
Comment (elorest) [reply src/amber/cli/commands/database.cr]: This is within the scope of the main `rescue` in run. Raising here is more object oriented as exiting is a way that we handle exceptions, not a replacement for exceptions.

File: src/amber/cli/commands/encrypt.cr (hunk 2 above)
Comment (elorest) [reply src/amber/cli/commands/encrypt.cr]: Also covered by existing rescue block. This is necessary for consistent messages. 

Again exiting isn't supposed to replace exceptions just handle them in a prettier way for the CLI.

**Pull request merged**